modules/gdnative/gdnative/gdnative.cpp

```diff
@@ -140,6 +141,10 @@ void GDAPI godot_print(const godot_string *p_message) {
 	print_line(*(String *)p_message);
 }
 
+void GDAPI godot_profiling_add_data(const char *p_signature, uint64_t p_time) {
```

Comment:
This function should go into modules/gdnative/nativescript/godot_nativescript.cpp

Reply:
Just applied this.